### ISSUE TITLE
Precise location from private receivers shows up on track map

### ISSUE THREAD

**Issue by j-sayers** [MotusDev/Motus-TO-DO#839]
This follows from the email thread earlier about the request by Project 203 to hide detections at a sensitive site...

I was just trying to see how the changes would affect what I saw when not logged in and changed Receiver deployment data to "user".

As excepted, there were no longer any receivers listed for the project under Explore Project. When I clicked through to their tags, detections that occurred at one of their (now temporarily private) receivers was referred to only by ID, and no lat lon was provided.

![2020-10-27_181452](https://user-images.githubusercontent.com/55001144/97368637-bd420380-1881-11eb-99bf-21dca142c6c3.png)

However, if viewed as a Map instead of Table, the pin point location was displayed. (not logged in)

![2020-10-27_181404](https://user-images.githubusercontent.com/55001144/97368640-c0d58a80-1881-11eb-857e-267dbb2f6247.png)

**Comment by Hussell** [comment]
The tracks are only recomputed once every 24 hours (a design decision I regret), so this should disappear soon.

**Comment by jsayers-bsc** [comment]
Ah, that would explain it then.

**Comment by StuMackenzie** [comment]
close?

**Comment by leberrigan** [comment]
Looks like the private receivers are now hidden on the map to public view, but the tagging site is still showing up:
https://motus.org/data/track?tagDeploymentId=24002

Is this supposed to be visible? Is there even a way to make tagging sites private?

**Comment by Hussell** [comment]
Confirmed for the initial example from project 203. The public tagging lat/long is pulled from vwData_tags_deployments_basic. To make them non-public, we have three options:
* Set the whole project to private (project.is_private_project = 1)
* Set the project's tags to private (project.is_private_tag = 1)
* Set the tags_deployments_basic securable for the project to disallow anyone outside the project from viewing tag species/date/location etc. (An administrator-only action on the project management page.)

We don't have any way to make some but not all tagging sites within a project private.

Project 203 had its securable for receiver locations set to being viewable by logged-in users only. I've set all its securables for tag deployments to the same level. This is still public, in the sense that anyone can make an account and log in, but the same solution seems to have been deemed sufficient for the receiver locations. OK?
